Lib/test/test_clinic.py

```diff
@@ -3294,7 +3294,15 @@ def test_meth_method_no_params(self):
         check = partial(self.assertRaisesRegex, TypeError, "no arguments")
         check(meth, 1)
         check(meth, a=1)
-        obj.test_meth_method_no_params_empty_kw_tuple()
+
+        from _testcapi import pyobject_vectorcall
+        pyobject_vectorcall(meth, None, None)
+        pyobject_vectorcall(meth, (), None)
+        pyobject_vectorcall(meth, (), ())
+        pyobject_vectorcall(meth, None, ())
+
+        check(pyobject_vectorcall, meth, (1,), None)
+        check(pyobject_vectorcall, meth, (1,), ("a",))
```

Comment:
```suggestion
        check(pyobject_vectorcall, meth, (), ("a",))
```

Reply:
That won't work.

Reply:
Ah right, the first tuple contains values.

Reply:
Addressed in eda7f03bb7a224b0b1fca821a8a88be90fa5cbc5